src/bufr/BufrParser/Exports/Export.cpp

```diff
@@ -30,7 +30,7 @@ namespace
 
         namespace Variable
         {
-            const char* Datetime = "datetime";
+            const char* Datetime = "dateTime";
```

Comment:
Why are we changing the name of datetime?

Reply:
This will require changes in every YAML file in existents...

Reply:
Every single YAML file already has to change - since we must have a units string attribute to make the data work in UFO or elsewhere.  And the new variable name is ```dateTime``` in place of ```datetime``` so I figured I would just do it all as needed and be consistent.

Reply:
The datatime tag, in this context, just refers to the class of object you are creating and doesn't have anything to do with what you might decide to call it in the end (by convention or otherwise). We should not conflate the two. I'm going to be very resistant to making arbitrary changes like this to the publicly facing interface.

Reply:
@rmclaren   Here is my rationale:  I've done the work of changing about 25 yaml files already.  Only those in branches not yet merged to develop would need changing and those would already need changes because we must enforce the proper use of ```units``` to accept this change in its entirety without backward compatibility.  Furthermore, a complete sweeping change with one letter capitalized will help to enforce that the variable name in the ioda output section must be changed and cannot be forgotten.  So, while unnecessary, I believe it is helpful for anyone in future either modifying or creating new yamls to get this right and consistent; otherwise, I can see confusion by forgetting with letter is capitalized.

Reply:
From a user's point-of-view, it makes better sense to me to keep general `datetime` in the `exports:` part. `dateTime` is a specific defined quantity to ioda-v2 convention.

Reply:
" I can see confusion by forgetting with letter is capitalized."

Yes exactly. `datetime` is like a built in keyword. Adding capitol letters not only makes it more likely that a user might spell it incorrectly, but it also makes it look like a user defined thing (usually camalCased in our case) as apposed to a built in thing which are lowercase words or conjunctions of words...